src/loader/mod.rs

```diff
@@ -52,6 +52,8 @@ pub enum Error {
     CommandLineCopy,
     /// Command line overflowed guest memory.
     CommandLineOverflow,
+    /// Device tree bianry too big.
```

Comment:
`aarch64`-specific, please tag with `cfg(target)` accordingly. This goes for all new additions.

Reply:
done.